bin/hdfcoinc/pycbc_multiifo_coinc_statmap

```diff
@@ -0,0 +1,472 @@
+#!/usr/bin/env python
+"""
+The program combines coincident output files generated
+by pycbc_multiifo_coinc_findtrigs to generated a mapping between SNR and FAP, along
+with producing the combined foreground and background triggers. It also has
+the capability of doing hierarchical removal of foreground triggers that are
+louder than all of the background triggers. We use this to properly assess
+the FANs of any other gravitational waves in the dataset.
+"""
+import argparse, h5py, itertools
+import lal, logging, numpy
+from pycbc.events import veto, coinc
+import pycbc.version, pycbc.pnutils, pycbc.io
+import sys
+
+def sec_to_year(sec):
+    return sec / lal.YRJUL_SI
```

Comment:
I think @ahnitz wants to avoid LAL constants, but this is fine with me. (Is this function coded elsewhere already though?)

Reply:
This function should be in conversions.py (if anywhere).  Will address either in this pull request or one immediately afterwards